lib/domain/LogStorage.js

```diff
@@ -33,6 +33,48 @@ class GelfLogStorage {
     this.file = `${namespace}/${id}.js`;
     this.namespace = namespace;
     this.fields = extractHTTPFields(req);
+    this.gelfClients = GelfLogStorage.prepareGelfClients();
+  }
+
+  static prepareGelfClients() {
+    let client;
+    const gelfClientList = [];
+
+    // support to old version of code. remove this when migrated.
+    if (Array.isArray(config.log.host)) {
+      for (let i = 0; i < config.log.host.length; i += 1) {
```

Comment:
In the new versions of ES, We can use `for of` operator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Iterating_over_an_Array

For example:
```js
for (let host of config.log.host) {
....
```